src/command/repo/create.c

```diff
@@ -19,10 +19,7 @@ cmdRepoCreate(void)
     MEM_CONTEXT_TEMP_BEGIN()
     {
         if (strEq(storageType(storageRepo()), STORAGE_S3_TYPE_STR))
-        {
-            storageS3Request(
-                (StorageS3 *)storageDriver(storageRepoWrite()), HTTP_VERB_PUT_STR, FSLASH_STR, NULL, NULL, true, false);
-        }
+            storageS3RequestP((StorageS3 *)storageDriver(storageRepoWrite()), HTTP_VERB_PUT_STR, FSLASH_STR);
```

Comment:
Basically, this got flipped. Previously, if the caller was not going to read the response in chunks (like StorageS3Read does) they had to set `.returnContent = true` to make the connection reusable, whether the content was important to them or not.

Now the content is automatically read into a buffer to make the connection resuable *unless* the caller sets `.contentIo = true` to indicate they will do the read in chunks themselves. Since it's rare for the caller to want to read in chunks (only one place in the S3 driver) this seems less prone to coding error.